config/initializers/devise.rb

```diff
@@ -8,6 +8,21 @@
 #
 # Use this hook to configure devise mailer, warden hooks and so forth.
 # Many of these configuration options can be set straight in your model.
+
```

Comment:
Optional:  basically did all these change please discard this file , so we do not have any conflict

Reply:
@brahimdidi Thanks for the review